musicbrainz/csv/get_genre.py

```diff
@@ -18,14 +18,13 @@
 for i in range(0, max_records, 50):
     PARAMS["offset"] = i
     resp = requests.get(url=URL, params=PARAMS, timeout=500)
-    temp = resp.json()
+    if resp.status_code == 503:
```

Comment:
So you’re only slowing down your requests once you hit the rate limit, despite knowing that you’re going to be rate limited?

Why not slow down all your requests to something that aligns with their documentation? 1/50th of a second if you want to be on the edge, 1/25th if you want a bit of room.